src/store/item-store.js

```diff
@@ -1,26 +1,23 @@
 import { Store } from 'consus-core/flux';
-
+import { searchItem } from '../lib/api-client.js';
 let item = null;
 
 class ItemStore extends Store {
     getItem() {
-        if (item === null) {
-            return item;
-        }
-        return {
-            id: item.id,
-            status: item.status
-        };
+        return item;
+    }
+    getAllItems(){
```

Comment:
This is really just a sanity thing for reading the code. When viewing all items I call this instead of getItem to imply that it can be more than one